src/main/java/org/ibp/api/java/impl/middleware/inventory/manager/LotServiceImpl.java

```diff
@@ -131,23 +140,41 @@ public String saveLot(final String programUUID,
 	@Override
 	public List<String> createLots(final String programUUID, final LotGeneratorBatchRequestDto lotGeneratorBatchRequestDto) {
```

Comment:
given the changes, we need to add unit tests

Reply:
I will do it.